specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/examples/GetModel.json

```diff
@@ -32,8 +32,19 @@
           "fields": [],
           "averageModelAccuracy": 0,
           "errors": []
-        }
+        },
+        "composedTrainResults": [
```

Comment:
Would it be possible to expand the returned JSON in this example to illustrate what field values would be returned?

Would it make sense to introduce a new type for composedTrainResults (not a TrainResult) that parallels the clusters collection in KeysResult more closely, in the sense that it returns a collection of fields and field accuracies per form type it can recognize?